config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  root "user#index"
+
+  resources :user, only: [:index, :show] do
+  end
```

Comment:
- [ ] After you create a posts controller, kindly add posts routes nested inside users routes as below: 

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
I did add post root

Reply:
Implemented 👍